src/client_reqwest.rs

```diff
@@ -102,44 +108,58 @@ impl AsyncTelegramApi for AsyncApi {
         Params: serde::ser::Serialize + std::fmt::Debug + std::marker::Send,
         Output: serde::de::DeserializeOwned,
     {
-        let json_string = crate::json::encode(&params)?;
-        let json_struct: Value = serde_json::from_str(&json_string).unwrap();
-        let file_keys: Vec<&str> = files.iter().map(|(key, _)| *key).collect();
-        let files_with_paths: Vec<(String, &str, String)> = files
-            .iter()
-            .map(|(key, path)| {
-                (
-                    (*key).to_string(),
-                    path.to_str().unwrap(),
-                    path.file_name().unwrap().to_str().unwrap().to_string(),
-                )
-            })
-            .collect();
-
-        let mut form = multipart::Form::new();
-        for (key, val) in json_struct.as_object().unwrap() {
-            if !file_keys.contains(&key.as_str()) {
-                let val = match val {
-                    Value::String(val) => val.to_string(),
-                    other => other.to_string(),
-                };
-
-                form = form.text(key.clone(), val);
+        #[cfg(not(target_arch = "wasm32"))]
+        {
+            use reqwest::multipart;
+            use serde_json::Value;
+            let json_string = crate::json::encode(&params)?;
+            let json_struct: Value = serde_json::from_str(&json_string).unwrap();
```

Comment:
Having an empty line in here splits the imports and code more clearly

```suggestion
            use reqwest::multipart;
            use serde_json::Value;

            let json_string = crate::json::encode(&params)?;
            let json_struct: Value = serde_json::from_str(&json_string).unwrap();
```

Reply:
I will resolve the issue, which seems to have been caused by numerous changes, by comparing it with the original file.